code/style.css

```diff
@@ -1,7 +1,188 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
```

Comment:
This file needs some work on it's structure! There are some empty rows that can be deleted and it would be awesome if the indention is the same everywhere. Then it will make the code more beautiful and more easy to read, which we always should aim for!